jvm/controls/src/main/resources/css/source-dialog.css

```diff
@@ -0,0 +1,36 @@
+@import "root.css"
+
+.source-dialog {
+    -fx-pref-width: 700px;
```

Comment:
should this be based on some percentage of the screen? What if 700px is too big on some screens?